api/docs/release.dox

```diff
@@ -142,10 +142,17 @@ changes:
  - No compatibility changes yet.
 
 Further non-compatibility-affecting changes include:
+ - Added support for statically linking clients and the DynamoRIO library into
+   a pure-static application via a new configure_DynamoRIO_static_client() CMake
+   function and new "_drstatic" static library CMake targets for the provided
+   extension libraries.
+ - Allow v2p.textproto file in a trace directory. This file is present in public traces.
+ - Allow v2p.textproto file to have one missing virtual_address field, which indicates
+   virtual_address == 0x0. Necessary in case a trace accesses virtual address 0x0.
  - Added the #dynamorio::drmemtrace::decode_cache_t library to make it easier and more
```

Comment:
This is now in the wrong list: 11.3 is already released so this can't go here.  It needs to be up on line 136.

Reply:
Oops. Apologies.